website/docs/cloud-docs/api-docs/reserved-tag-keys.mdx

```diff
@@ -0,0 +1,271 @@
+
+Properties without a default value are required.
+
+| Key path | Type | Default | Description |
+|---|---|---|--- |
+| `data.type` | string | none | Must be `reserved-tag-keys`. |
+| `data.attributes.key` | string | none | The key targeted by this reserved
+tag key. |
+| `data.attributes.disable-overrides` | boolean | none | If `true`, disables
+overriding inherited tags with the specified key at the workspace level. |
+
+### Sample payload
+
+```json
+{
+  "data": {
+    "type": "reserved-tag-keys",
+    "attributes": {
+      "key": "environment",
+      "disable-overrides": false
+    }
+  }
+}
+```
+
+### Sample request
+
+```shell-session
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  https://app.terraform.io/api/v2/organizations/${ORGANIZATION_NAME}/reserved-tag-keys
+```
+
+### Sample response
+
+```json
+{
+    "data": {
+        "id": "rtk-Tj86UdGahKGDiYXY",
+        "type": "reserved-tag-keys",
+        "attributes": {
+            "key": "environment",
+            "disable-overrides": false,
+            "created-at": "2024-09-04T05:02:06.794Z",
+            "updated-at": "2024-09-04T05:02:06.794Z"
+        }
+    }
+}
+```
+
+## Update a reserved tag key
+
+`PATCH /reserved-tags/:reserved_tag_key_id`
+
+This PATCH endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required.
+
+| Key path | Type | Default | Description |
+|---|---|---|--- |
+| `data.type` | string | none | Must be `reserved-tag-keys`. |
+| `data.attributes.key` | string | none | The key targeted by this reserved
+tag key. |
+| `data.attributes.disable-overrides` | boolean | none | If `true`, disables
+overriding inherited tags with the specified key at the workspace level. |
+
+### Sample payload
+
+```json
+{
+  "data": {
+    "id": "rtk-Tj86UdGahKGDiYXY",
+    "type": "reserved-tag-keys",
+    "attributes": {
+      "key": "env",
+      "disable-overrides": true,
+      "created-at": "2024-09-04T05:02:06.794Z",
+      "updated-at": "2024-09-04T05:02:06.794Z"
+    }
+  }
+}
+```
+
+### Sample request
+
+```shell-session
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request PATCH \
+  https://app.terraform.io/api/v2/reserved-tags/${RESERVED_TAG_ID}
+```
+
+### Sample response
+
+```json
+{
+    "data": {
+        "id": "rtk-zMtWLDftAjY3b5pA",
+        "type": "reserved-tag-keys",
+        "attributes": {
+            "key": "env",
+            "disable-overrides": true,
+            "created-at": "2024-09-04T05:05:10.449Z",
+            "updated-at": "2024-09-04T05:05:13.486Z"
+        }
+    }
+}
+```
+
+## Delete a reserved tag key
+
+`DELETE /reserved-tags/:reserved_tag_key_id`
+
+### Sample payload
+
+This endpoint does not require a payload.
+
+### Sample request
+
+```shell-session
+curl \
```

Comment:
there are several of these where we're missing `$` in front of the command for the sample request. not a blocker but just wanted to call it out!

````suggestion
```shell-session
$ curl \
````

Reply:
Addressed in [2578674](https://github.com/hashicorp/terraform-docs-common/pull/699/commits/25786745db079467e4ffa27cb9e530a00da6c840).